azurelinuxagent/common/version.py

```diff
@@ -116,6 +117,32 @@ def get_distro():
     return osinfo
 
 
+def get_lis_version():
+    """
+    This uses the Linux kernel's 'modinfo' command to retrieve the
+    "version" field for the "hv_vmbus" kernel module (the LIS
+    drivers). This is the documented method to retrieve the LIS
+    drivers' version. Every Linux guest on Hyper-V will have this
+    module (and so this version field).
+
+    This telemetry can be moved to a separate extension when the Linux
+    Integration Services team (@lisval) develops such.
```

Comment:
@narrieta already done!

Reply:
Please replace
    This telemetry can be moved to a separate extension when the Linux
    Integration Services team (@lisval) develops such.

With something similar to
    This function is used to generate telemetry for the version of the LIS drivers installed on the VM. The function and associated telemetry can be removed after a few releases.

Reply:
Done.